app/services/katello/pulp/repository.rb

```diff
@@ -120,6 +120,56 @@ def mirror_importer_connection_options
       def proxy_host_importer_value
         root.ignore_global_proxy ? "" : nil
       end
+
+      def refresh
+        output = []
+        output << update_or_associate_importer
+        output << update_or_associate_distributors
+        output << remove_unnecessary_distributors
+      end
```

Comment:
we can't save this output onto the task, and i get an error when running from a task:

```
NoMethodError

undefined method `to_hash' for #<Runcible::Models::IsoDistributor:0x000000000cd41c58> Did you mean? to_s
```
we may just not save anything on the output to the task.